### PR TITLE
Ensuring USE_DB_AUTHENTICATION is true in single URL scraper.

### PR DESCRIPTION
**Pull request by kevinswiber**
When `USE_DB_AUTHENTICATION` is set to `false` in an environment variable, a boolean check for `process.env.USE_DB_AUTHENTICATION` will always return `true`, as `process.env` values are strings.  This PR fixes it.

Bug introduced in 01fab6e036f44406204cc7ade38116654120a19f.

### CODE REVIEW

**Review by rafaelsideguide — COMMENTED** [review]
Makes sense. `USE_DB_AUTH = "false"` evaluates to `true`. @nickscamara I think we should find a better way to define the scraping strategies. Maybe directly in `.env` file?

**Review by rafaelsideguide — APPROVED** [review]

File: apps/api/src/scraper/WebScraper/single_url.ts [hunk 1]
```diff
@@ -85,18 +89,18 @@ function getScrapingFallbackOrder(
...
+    useScrapingBee ? "scrapingBeeLoad" : undefined,
+    useDatabaseAuth ? undefined : "playwright",
     "fetch",
   ].filter(Boolean);
 
   if (isWaitPresent || isScreenshotPresent || isHeadersPresent) {
     defaultOrder = [
       "fire-engine",
-      process.env.USE_DB_AUTHENTICATION ? undefined : "playwright",
+      useDatabaseAuth ? undefined : "playwright",
```
Comment (kevinswiber) [inline apps/api/src/scraper/WebScraper/single_url.ts]: I do find this a little confusing. Why would we not use Playwright when there's database auth enabled?

Comment (rafaelsideguide) [reply apps/api/src/scraper/WebScraper/single_url.ts]: Agreed. I've switched to `useFireEngine` instead. No need to check the DB here—using both FireEngine and Playwright is kind of redundant. They both use similar scraping methods, so if one doesn’t work, the other probably won’t either.

**Comment by yekkhan** [comment]
I wanted to kindly inquire about the status of the pull request [#516 ]. Could you let me know when it might be reviewed or merged?

**Review by nickscamara — APPROVED** [review]

**Comment by nickscamara** [comment]
@yekkhan merging now!

**Pull request merged**